### PR TITLE
[Keymap] for sharkby7e

### PR DESCRIPTION
**Pull request by sharkby7e**
## Types of Changes

- [X] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/sharkby7e/config.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+//#define TAPPING_FORCE_HOLD
+#define TAPPING_TERM 200
```
Comment (waffle87) [inline keyboards/crkbd/keymaps/sharkby7e/config.h:32]:
```suggestion
```

File: keyboards/crkbd/keymaps/sharkby7e/rules.mk [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# MOUSEKEY_ENABLE   = yes    # Mouse keys
+RGBLIGHT_ENABLE   = yes    # Enable WS2812 RGB underlight.
+# OLED_ENABLE       = yes
+# OLED_DRIVER       = SSD1306
+LTO_ENABLE        = yes
+BOOTLOADER        = caterina
```
Comment (waffle87) [inline keyboards/crkbd/keymaps/sharkby7e/rules.mk:6]:
```suggestion
RGBLIGHT_ENABLE = yes
BOOTLOADER = caterina
```

**Review by waffle87 — CHANGES_REQUESTED** [review]
There are also merge conflicts that need to be resolved.

**Review by sharkby7e — COMMENTED** [review]
accepted ur changes, then fixed things on my end

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**